lib/adapter/freshheads/CssAdapter.ts

```diff
@@ -43,7 +49,7 @@ export default class CssAdapter implements Adapter {
         const isProduction = builderConfig.env === Environment.Production;
 
         const rule: RuleSetRule = {
-            test: /\.css$/,
+            test: /\.s?css$/,
```

Comment:
Moet deze rule dan niet afhankelijk zijn van de config? Dus `.css$` als css, en als Sass ook aan staat `.scss$`? Anders krijg je misschien vreemde dingen?

Reply:
ik denk niet dat je echt vreemde dingen gaat krijgen want als je sass uit het staan krijg je sowieso de melding dat er geen loader is voor dat type. Maar aangezien we de setting hebben kan het wel wat verduidelijking bieden. Dus zal het wel meenemen.